### DISCUSSION TITLE
How to debug which wheel platform tag matches the current platform?

### DISCUSSION THREAD

**Discussion by sanmai-NL** [pdm-project/pdm#2336] [Q&A] [answered]
In the resolution logs (`-v`) I only see what wheel variants are encountered and which ones match, if any one does. Is there a way to log what platform tag PDM is looking for, e.g., as part of debug info output by `pdm info`?

**Comment by sanmai-NL** [comment]
```
+ pdm info --env
{
  "implementation_name": "cpython",
  "implementation_version": "3.11.6",
  "os_name": "posix",
  "platform_machine": "aarch64",
  "platform_release": "6.5.6-200.fc38.aarch64",
  "platform_system": "Linux",
  "platform_version": "#1 SMP PREEMPT_DYNAMIC Fri Oct  6 19:34:05 UTC 2023",
  "python_full_version": "3.11.6",
  "platform_python_implementation": "CPython",
  "python_version": "3.11",
  "sys_platform": "linux"
}
```

I see `platform_machine`, but not information that indicates that the [PEP 656 `musllinux` platform tag](https://peps.python.org/pep-0656/) is applicable.

**Reply by frostming** [reply, marked as answer]
We don't provide such information in any output or log. But you can get it by `project.environment.target_python.supported_tags()`

**Reply by sanmai-NL** [reply]
Okay, I don't know how to execute such API calls against PDM. Or do you mean, we could enhance PDM itself? Would you care for a PR that does log the supported tags?
